scripts/restore/README.md

```diff
@@ -1,30 +1,64 @@
 ### Restore Harbor
-With the script `restore-harbor.sh` you can restore the database in Harbor from a backup in S3.
+With the k8s job you can restore the database in Harbor from a backup in S3.
+The steps should be preformed from the `compliantkubernetes-apps` root directory.
+*Note this k8s job does not include restore from gcs.*
 
-Before restoring the database, make sure that Harbor is installed. It can be installed normally.
+Before restoring the database, make sure that Harbor is installed.
+It can be installed normally.
 
-In order to run the restore script you need the aws client and the postgres client installed:
-```bash
-curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
-unzip awscli-bundle.zip
-sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
-sudo apt install postgresql-client-10
-sudo apt install postgresql-client-common
-```
-Set these variables for the restore-script:
+#### Env variables
+Set these variables for the restore job:
 ```bash
 export CK8S_CONFIG_PATH=<your config path>
 export S3_BUCKET=$(yq read "${CK8S_CONFIG_PATH}/sc-config.yaml" objectStorage.buckets.harbor)
 export S3_REGION_ENDPOINT=$(yq read "${CK8S_CONFIG_PATH}/sc-config.yaml" objectStorage.s3.regionEndpoint)
-export AWS_ACCESS_KEY_ID=$(sops -d "${CK8S_CONFIG_PATH}/secrets.yaml" | yq read - objectStorage.s3.accessKey)
-export AWS_SECRET_ACCESS_KEY=$(sops -d "${CK8S_CONFIG_PATH}/secrets.yaml" | yq read - objectStorage.s3.secretKey)
 ```
-While restoring we need to stop all harbor pods except for the database. Then restart them after the restoration.
-Run these commands to stop the pods, restore the database, and restart the pods:
+
+##### Optional env variables
+
+The job will restore the latest backup.
+`$SPECIFIC_BACKUP` can be used to specify which backup.
+To get a list of available backups use:
+```
+aws s3 ls ${S3_BUCKET}"/backups" --recursive --endpoint-url=${S3_REGION_ENDPOINT}
+```
+Then set:
+```bash
+export SPECIFIC_BACKUP=<backups/xxxxxxxxxx.sql.gz> # Optional
+```
+
+#### Restore
+
+Setup the necessary job.yaml and configmap:
+
 ```bash
+envsubst > tmp-job.yaml < scripts/restore/restore-harbor-job.yaml
+./bin/ck8s ops kubectl sc create configmap -n harbor restore-harbor --from-file=scripts/restore/restore-harbor.sh
```

Comment:
Could we use yq instead of envsubst?

Reply:
What is the benefits of using yq here instead of envsubst?

Reply:
Envsubst is kind of unpredictable if you aren't careful as it will replace any `$` it finds in the file.
It this case it probably fine

Reply:
Ah yeah well that is true. Yes should be fine in this case.